qutebrowser/commands/runners.py

```diff
@@ -66,6 +66,7 @@ def replace_variables(win_id, arglist):
         'url:host': lambda: _current_url(tabbed_browser).host(),
         'clipboard': utils.get_clipboard,
         'primary': lambda: utils.get_clipboard(selection=True),
+        'link_hovered': lambda: tabbed_browser._now_focused._last_hovered_link,
```

Comment:
You're accessing private variables here - `last_hovered_link` should be public in `TabData`. As for `tabbed_browser._now_focused`, I think you can use `tabbed_browser.widget.currentWidget()` instead.